### PR TITLE
Misc. improvements

### PR DESCRIPTION
**Pull request by rf-**
See individual commits for details.

### CODE REVIEW

File: css/crosswordnexus.css [hunk 1]
```diff
@@ -535,69 +541,59 @@ input.cw-hidden-input {
...
-        display: none;
-    }
-    .cw-buttons-holder {
-        height: 45px;
-        padding: 0 10px;
-    }
-    .cw-top-text-wrapper {
-        height: 40px;
-    }
+.cw-max-width-600.cw-main {
```
Comment (boisvert42) [inline css/crosswordnexus.css:577]:
```suggestion
.cw-max-width-600 .cw-main {
```

File: js/crosswords.js [hunk 2]
```diff
@@ -1113,54 +1140,49 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
...
           <div class="modal-header">
-            <span class="close" id="modalClose">&times;</span>
+            <span class="modal-close">&times;</span>
             <span class="modal-title">${title}</span>
           </div>
           <div class="modal-body">
             ${content}
           </div>
           <div class="modal-footer">
-            <button id="modal-button" class="modal-button">${button_text}</button>
```
Comment (boisvert42) [inline js/crosswords.js:1151]: Is there a particular reason to remove the `id="modal-button"` here?  It's broken the ability to close the modal by clicking on the "close" button.  If ids are bad we could add a new class and change the code on line 1199 accordingly.

File: js/crosswords.js [hunk 3]
```diff
@@ -1113,54 +1140,49 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
...
+            <button class="cw-button">${button_text}</button>
```
Comment (boisvert42) [inline js/crosswords.js]:
```suggestion
            <button class="cw-button" id="modal-button">${button_text}</button>
```
unless this is a bad idea for some reason?

Comment (rf-) [reply js/crosswords.js]: No reason, thanks! IDs kind of suck but it’s really not a big deal in this case, just habit

**Review by boisvert42 — APPROVED** [review]
looks good

**Pull request merged**